devices.js

```diff
@@ -942,6 +967,15 @@ const devices = [
         fromZigbee: [fz.xiaomi_battery_3v, fz.DJT11LM_vibration],
         toZigbee: [tz.DJT11LM_vibration_sensitivity],
     },
+    {
+        zigbeeModel: ['lumi.vibration.agl01'],
+        model: 'DJT12LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara vibration sensor',
+        supports: 'drop, tilt and touch',
+        fromZigbee: [fz.DJT12LM_vibration],
+        toZigbee: [tz.DJT11LM_vibration_sensitivity],
```

Comment:
Does setting the sensitivity work?

Reply:
setting the sensitivity can **not**  work properly , So removed and remain for someone else to commit for this.